### ISSUE TITLE
General checks for consistency between exchange catalogues and resources in exchange sets

### ISSUE THREAD

**Issue by rmalyankar** [iho-ohi/S-100-Validation-Checks#24]
General checks for consistency between discovery metadata blocks in CATALOG.XML and the respective resource (dataset, support file, catalogue).

(To be developed. I think issues #11 and the Dataset/CATALOG.XML checks and feedback on them mentioned in #13 can be a starting point.)

**Comment by rmalyankar** [comment]
Various checks are included in the Part 17 checks in the Edition 1.0.0 list, including:

- S100_Dev0315
- S100_Dev0580
- S100_Dev0155
- S100_Dev0586
- S100_Dev0553
- S100_Dev0554
- S100_Dev0574
- S100_Dev0570
- S100_Dev0160 as revised in #82 (pertaining to consistent metadata)

This issue will be kept open for the time being as a place to request additional checks for consistency between discovery metadata blocks and resources. Alternatively, they can be raised as new issues (metadata consistency discussion should continue in #82).

**Comment by LizHahessy** [comment]
Agreed at VTC 13 17/06/25 to close for Eidtion 1.0.0, can be picked up under issue #82 or open a new issue if needed.
